requirements/env_climada.yml

```
@@ -38,6 +38,6 @@ dependencies:
   - tabulate>=0.9
   - tqdm>=4.66
   - unittest-xml-reporting>=3.2
-  - xarray>=2024.1,<2024.2  # 2024.2 introduced an unhandled AttributeError in Hazard.from_xarray_raster
+  - xarray>=2024.2
```

Why raise to 2024.2? We are now compatible to it but we should still be compatible to 2024.1 🤔 
```suggestion
  - xarray>=2024.1
```

---

true but sooner or later the version will be updated to whatever is current
and this would make it easier to adapt to the new xarray. i suspect it to slow down "something" and i'm not sure whether we want that. 🤔

---

Sure, we have to adapt to later versions, and this is what we do here. Also, the GitHub pipeline always tries to install the latest possible version. I am just a bit hesitant to immediately block any older versions. But go ahead! If something breaks, we can revert.